lib/cli/commands/channelbalance.ts

```diff
@@ -1,6 +1,39 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ChannelBalanceRequest } from '../../proto/xudrpc_pb';
+import Table, { HorizontalTable } from 'cli-table3';
+import colors from 'colors/safe';
+import { ChannelBalanceRequest, ChannelBalanceResponse } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  colors.red('Ticker'),
+  colors.red('balance'),
+  colors.red('Pending open balance'),
```

Comment:
Maybe just "Pending Balance" - I don't think "open" is really necessary here and space is limited.